src/eterna/UndoBlock.ts

```diff
@@ -19,8 +19,8 @@ import {TargetType} from './puzzle/Puzzle';
 export interface FoldData {
     folderName_: string;
     sequence_: number[];
-    pairs_array_: Map<boolean, number[][]>;
-    params_array_: Map<boolean, Param[][]>;
+    pairs_array_: [boolean, number[][]][] | number[][];
+    params_array_: [boolean, Param[][]][] | Param[][];
```

Comment:
I see, reading this in conjunction with the changes around ~130, this is for serialization.